Resources/Prototypes/_Backmen/Entities/Mobs/NPCs/terror_spiders.yml

```
@@ -0,0 +1,374 @@
+    allowedStates:
+    - Alive
+    damageCap: 89
+    damage:
+      types:
+        Poison: -0.07
+      groups:
+        Brute: -0.07
+        Burn: -0.07
+  - type: NpcFactionMember
+    factions:
+      - Xeno
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)